src/video.js

```diff
@@ -1,8 +1,35 @@
 import { videoAdapters } from './adaptermanager';
+import { getBidRequest, deepAccess } from './utils';
+
+const VIDEO_MEDIA_TYPE = 'video';
+const INSTREAM = 'instream';
+const OUTSTREAM = 'outstream';
 
 /**
  * Helper functions for working with video-enabled adUnits
  */
-export const videoAdUnit = adUnit => adUnit.mediaType === 'video';
+export const videoAdUnit = adUnit => adUnit.mediaType === VIDEO_MEDIA_TYPE;
 const nonVideoBidder = bid => !videoAdapters.includes(bid.bidder);
-export const hasNonVideoBidder = adUnit => adUnit.bids.filter(nonVideoBidder).length;
+export const hasNonVideoBidder = adUnit =>
+  adUnit.bids.filter(nonVideoBidder).length;
+
+/*
+ * Validate that the assets required for video context are present on the bid
+ */
+export function videoBidIsValid(bid) {
+  const bidRequest = getBidRequest(bid.adId);
+  const context =
+    bidRequest && deepAccess(bidRequest, 'mediaTypes.video.context');
+
+  // if context not defined assume default 'instream' for video bids
+  if (!bidRequest || !context || context === INSTREAM) {
+    return !!(bid.vastUrl || bid.vastPayload);
```

Comment:
We are removing`bid.descriptionUrl` when prebid-cache is live or stable.
So on that note, will this also include `bid.descriptionUrl` ? Not very sure though.

Reply:
It won't, just `bid.vastUrl` or a vast payload is required. `vastPayload` was renamed `vastXml`, will push an updated to fix that here